### PR TITLE
add support of viddy

### LINKED ISSUE #794: Add Support of viddy
Add installation support for https://github.com/sachaos/viddy cli
## Expected Behaviour

Add installation support for https://github.com/sachaos/viddy
## Current Behaviour

Not available
## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [x] No

### PR DESCRIPTION
**Pull request by nitishkumar71**
Signed-off-by: Nitishkumar Singh <nitishkumarsingh71@gmail.com>

## Description

add [viddy](https://github.com/sachaos/viddy) cli installation support in arkade
## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Closes #794 

## How Has This Been Tested?

1. Written test cases accordingly
2. Get viddy CLI from local build `arkade get viddy`
```
Downloading: viddy
2022/10/23 19:50:57 Looking up version for viddy
2022/10/23 19:50:58 Found: v0.3.6
Downloading: https://github.com/sachaos/viddy/releases/download/v0.3.6/viddy_0.3.6_Linux_x86_64.tar.gz
3.29 MiB / 3.29 MiB [-------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
/tmp/viddy_0.3.6_Linux_x86_64.tar.gz written.
2022/10/23 19:51:02 Looking up version for viddy
2022/10/23 19:51:02 Found: v0.3.6
2022/10/23 19:51:02 Extracted: /tmp/viddy
2022/10/23 19:51:02 Copying /tmp/viddy to /home/nitishkumar/.arkade/bin/viddy

Wrote: /home/nitishkumar/.arkade/bin/viddy (8.286MB)

# Add arkade binary directory to your PATH variable
export PATH=$PATH:$HOME/.arkade/bin/

# Test the binary:
/home/nitishkumar/.arkade/bin/viddy

# Or install with:
sudo mv /home/nitishkumar/.arkade/bin/viddy /usr/local/bin/
```
3. Test viddy installation `viddy -v`
```
viddy version: 0.3.6
```
## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/get/tools.go [hunk 1]
```diff
@@ -2967,5 +2967,33 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "sachaos",
+			Repo:        "viddy",
+			Name:        "viddy",
+			Description: "Validate your Kubernetes configuration files, supports multiple Kubernetes versions",
```
Comment (alexellis) [inline pkg/get/tools.go:2975]: Where is the description from?

It should be:

A modern watch command. Time machine and pager etc.


https://github.com/sachaos/viddy

File: pkg/get/get_test.go [hunk 2]
```diff
@@ -5168,3 +5168,61 @@ func Test_DownloadKubeval(t *testing.T) {
...
+			url:     "https://github.com/sachaos/viddy/releases/download/v0.3.6/viddy_0.3.6_Linux_arm64.tar.gz",
+		},
+		{
+			os:      "ming",
+			arch:    arch64bit,
+			version: version,
+			url:     "https://github.com/sachaos/viddy/releases/download/v0.3.6/viddy_0.3.6_Windows_x86_64.tar.gz",
+		},
+		{
+			os:      "ming",
```
Comment (alexellis) [inline pkg/get/get_test.go:5210]: Please remove 64-bit windows.

Comment (nitishkumar71) [reply pkg/get/get_test.go:5210]: Sorry didn't get the issue

Comment (nitishkumar71) [reply pkg/get/get_test.go]: Please ignore

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**